hazelcast/src/main/java/com/hazelcast/internal/util/RuntimeAvailableProcessors.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.util;
+
+/**
+ * Utility to access {@link Runtime#availableProcessors()} and optionally override its return value.
```

Comment:
Can a line of comment be added that the sole purpose of this class it to make the number of available processors configurable for the sake of testing. And that override/resetoverride should only be used for testing purposes.